darwin/importer/importer.py

```diff
@@ -144,8 +144,12 @@ def maybe_console(*args: Union[str, int, float]) -> None:
         parsed_files = list(map(importer, tqdm(files) if is_console else files))
 
     maybe_console("Finished.")
-
-    if not isinstance(parsed_files, list):
+    # Sometimes we have a list of lists of AnnotationFile, sometimes we have a list of AnnotationFile
+    # We flatten the list of lists
+    if isinstance(parsed_files, list):
+        if isinstance(parsed_files[0], list):
```

Comment:
if not doing anything else with the inner conditional, can flatten this, python supports conditional short circuiting
```python
if isinstance(parsed_files, list) and isinstance(parsed_files[0], list):
    ...
```
This won't raise if parsed_files isn't indexable because the first condition will short circuit.

Reply:
This change actually causes these tests to fail for some reason:

```
FAILED tests/darwin/importer/importer_mcpu_test.py::Test_find_and_parse::test_runs_single_threaded_if_use_multi_cpu_false - Asse...
FAILED tests/darwin/importer/importer_mcpu_test.py::Test_find_and_parse::test_uses_mpire_if_use_multi_cpu_true - AssertionError:...
```

Going to merge without it, can't spend any longer on this ticket.